examples/tutorials/asr_inference_with_ctc_decoder_tutorial.py

```diff
@@ -161,26 +162,27 @@
 # Construct Beam Search Decoder
 # -----------------------------
 #
-# The decoder can be constructed using the
-# :py:func:`torchaudio.prototype.ctc_decoder.kenlm_lexicon_decoder`
-# factory function.
-# In addition to the previously mentioned components, it also takes in
-# various beam search decoding parameters and token/word parameters.
+# The decoder can be constructed using the ``kenlm_lexicon_decoder``
+# factory function from ``torchaudio.prototype.ctc_decoder``. In addition
+# to the previously mentioned components, it also takes in various beam
+# search decoding parameters and token/word parameters. The full list of
+# parameters can be found
+# `here <https://pytorch.org/audio/main/prototype.html#kenlm-lexicon-decoder>`__.
```

Comment:
```suggestion
# :py:func:`here <torchaudio.prototype. kenlm_lexicon_decoder>`.
```

Since once this tutorial is released, then we will have multiple versions of this tutorials, so I think it's better to use interlink.